src/tutorials/building-dapps-for-quorum-private-enterprise-blockchains.md

```diff
@@ -141,7 +142,7 @@ Now that we have Truffle set up, we can move onto code.
 
 ## Deploying smart contracts on Quorum
 
-We won't spend too much time talking about writing or deploying contracts in Truffle since we have [ample documentation](/docs), however we do want to show how deploying contracts applies to Quorum.
+We won't spend too much time talking about writing or deploying contracts in Truffle since we have [sample documentation](/docs), however we do want to show how deploying contracts applies to Quorum.
```

Comment:
Think this was meant to be ample

```suggestion
We won't spend too much time talking about writing or deploying contracts in Truffle since we have [ample documentation](/docs), however we do want to show how deploying contracts applies to Quorum.
```